### PR TITLE
Add sign-up: how did you hear about Trustroots? #879

### PR DESCRIPTION
**Pull request by RGHuynh**
WIP

* Add input box "how the you hear about Trustroots" to signup
* Send user responses to InfluxDB

Fixes #
issue #879

### CODE REVIEW

**Comment by petr-hajek** [comment]
@RGHuynh Hey, planning to continue working on this?

File: modules/users/server/controllers/users.authentication.server.controller.js [hunk 1]
```diff
@@ -84,6 +84,20 @@ exports.signup = function (req, res) {
 
     },
 
+    // Report how did you hear about us to stats
+    function () {
```
Comment (guaka) [inline modules/users/server/controllers/users.authentication.server.controller.js]: I'm no async wizard ✨  either but it Looks like this function should receive a callback var `done`, which then should be called in `statService.stat`'s callback, I'm testing this now in #970.

**Comment by RGHuynh** [comment]
> @RGHuynh Hey, planning to continue working on this?

Yeah, I'm just stuck where i can't figure out how to send the user response to influxDB

**Comment by guaka** [comment]
continue with #970

**Pull request closed without merging**